pkg/resources/storage_integration_test.go

```diff
@@ -131,6 +154,26 @@ func expectReadStorageIntegration(mock sqlmock.Sqlmock) {
 	mock.ExpectQuery(`DESCRIBE STORAGE INTEGRATION "test_storage_integration"$`).WillReturnRows(descRows)
 }
 
+func expectReadStorageIntegrationWithS3GOV(mock sqlmock.Sqlmock) {
+	showRows := sqlmock.NewRows([]string{
+		"name", "type", "category", "enabled", "created_on"},
+	).AddRow("test_storage_integration_with_s3gov", "EXTERNAL_STAGE", "STORAGE", true, "now")
+	mock.ExpectQuery(`^SHOW STORAGE INTEGRATIONS LIKE 'test_storage_integration_with_s3gov'$`).WillReturnRows(showRows)
+
+	descRows := sqlmock.NewRows([]string{
+		"property", "property_type", "property_value", "property_default",
+	}).AddRow("ENABLED", "Boolean", true, false).
```

Comment:
minor comment - could you reformat the lines to be aligned like the following?

```
.AddRow("ENABLED", "Boolean", true, false)
.AddRow("STORAGE_PROVIDER", "String", "S3GOV", nil)
.AddRow("STORAGE_ALLOWED_LOCATIONS", "List", "s3://bucket-a/path-a/,s3://bucket-b/", nil)
.AddRow("STORAGE_BLOCKED_LOCATIONS", "List", "s3://bucket-c/path-c/,s3://bucket-d/", nil)
...
.AddRow("STORAGE_AWS_EXTERNAL_ID", "String", "AGreatExternalID", nil)
```

Reply:
sure